packages/vscode-extension/package.json

```diff
@@ -93,7 +103,16 @@
 				"key": "alt+shift+u",
 				"when": "resource =~ //FileCabinet/(SuiteScripts|Templates/(E-mail%20Templates|Marketing%20Templates)|Web%20Site%20Hosting%20Files|SuiteApps)/.+$/"
 			}
-		]
+		],
+		"menus": {
+			"explorer/context": [
+				{
+					"when": "resource =~ //FileCabinet/(SuiteScripts|Templates/(E-mail%20Templates|Marketing%20Templates)|Web%20Site%20Hosting%20Files|SuiteApps)/.+$/",
```

Comment:
This "regex" will match a child directory in `FileCabinet/SuiteScript/ChildDir` and by the name of the command I think it shouldn't.

Reply:
Hey, David, I don't understand. Import files should allow only importing files from Suitescript Templates and WebSiteHostingFiles folders, and all of their childs... I don't see the problem in this regex.